src/layout/header/Header.tsx

```diff
@@ -107,18 +109,20 @@ export const Header = () => {
             <>
               {customer?.shortName &&
                 (user?.isEditor ? (
-                  <Button
-                    sx={{ whiteSpace: 'nowrap', borderRadius: '2rem' }}
```

Comment:
Nå er det ingen indikator på om denne er klikkbar eller ikke, er det det som er ønskelig?
Husker jeg for en tid tilbake fikk en oppgave som bare handlet om å legge til border for å skille på om det er en knapp eller bare info